### PR TITLE
PR: Set English as default language if getdefaultlocale gives errors

### LINKED ISSUE #3627: Spyder won't start because of "ValueError: unknown locale: UTF-8" in macOS
##I cannot get my spyder app running.
## I installed IRAF yesterday and ever since spyder won't start when I type the command in the terminal.
##Here's a screenshot of what i get:
screen shot 2016-10-31 at 7 10 42 pm

### PR DESCRIPTION
**Pull request by malliwi88**
Fixes #3627.

### CODE REVIEW

**Comment by malliwi88** [comment]
@ccordoba12 

Already, I achieved fix my problem. Only need to wait for the checks of PR on remote repo ;)

File: spyder/config/base.py [hunk 1]
```diff
@@ -267,18 +267,21 @@ def get_interface_language():
     """
     locale_language = locale.getdefaultlocale()[0]
```
Comment (ccordoba12) [inline spyder/config/base.py]: William, the try except needs to be around this line, not below :-)

Besides, you only need to catch `ValueError`, which was the error reported on issue #3627.

**Comment by ccordoba12** [comment]
I'm glad you solved your problems :-) At last!

File: spyder/config/base.py [hunk 2]
```diff
@@ -265,9 +265,13 @@ def get_interface_language():
     2.) Spyder provides ('en',  'fr', 'es' and 'pt_BR'), if the locale is
     either 'pt' or 'pt_BR', this function will return 'pt_BR'
     """
-    locale_language = locale.getdefaultlocale()[0]
 
-    language = DEFAULT_LANGUAGE
+    # Solves issue #3627
+    try:
+        locale_language = locale.getdefaultlocale()[0]
+
```
Comment (ccordoba12) [inline spyder/config/base.py]: Please remove this blank line.

**Comment by ccordoba12** [comment]
Thanks @malliwi88, just a minor correction and this one is ready :-)

File: spyder/config/base.py (hunk 2 above)
Comment (malliwi88) [inline spyder/config/base.py]: Ok, I just done it. waiting for test. :+1:

File: spyder/config/base.py [hunk 3]
```diff
@@ -280,6 +283,8 @@ def get_interface_language():
                 language = lang
                 break
 
+    language = DEFAULT_LANGUAGE
```
Comment (ccordoba12) [inline spyder/config/base.py]: Please place this line after the try/except you just added (as it was before).

File: spyder/config/base.py [hunk 4]
```diff
@@ -265,7 +265,12 @@ def get_interface_language():
     2.) Spyder provides ('en',  'fr', 'es' and 'pt_BR'), if the locale is
     either 'pt' or 'pt_BR', this function will return 'pt_BR'
     """
-    locale_language = locale.getdefaultlocale()[0]
+
+    # Solves issue #3627
```
Comment (ccordoba12) [inline spyder/config/base.py]: Nice! Thanks for adding this comment :-)

**Pull request merged**